content/en/blogs/porter-in-cncf-landscape.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: 'Porter: An Innovative Cloud Native Service Proxy in CNCF Landscape'
+keywords: Kubernetes, bare-metal, LoadBalancer, Porter
+description: Porter allows you to create Kubernetes services of type LoadBalancer in bare metal cluster, which makes you enjoy the consistent experience with the cloud. Porter has been accepted into CNCF Landscape as a promising newcomer.
 tag: 'Load Balancer, Kubernetes, Porter'
```

Comment:
@Sherlock113 Please note that we need to add `keywords` and `description` for all markdown in the website. If you find any blogs missing these two parts, you can add them accordingly.